frontend/src/metabase/query_builder/components/template_tags/TagEditorParamParts/FieldMappingSelect.tsx

```diff
@@ -36,7 +33,11 @@ export function FieldMappingSelect({
     <InputContainer>
       <ContainerLabel>
         {t`Field to map to`}
-        {tag.dimension == null && <ErrorSpan>{t`(required)`}</ErrorSpan>}
+        {tag.dimension == null && (
+          <Text c="error" span={true} ml="sm">
```

Comment:
I was hoping we're going to fix it with whitespace (not margin) as in other places ([ParameterSettings](https://github.com/metabase/metabase/blob/35d9a8177edd2885bcfa95107c34201c9691bf35/frontend/src/metabase/parameters/components/ParameterSettings/ParameterSettings.tsx#L243), [DefaultRequiredValueControl](https://github.com/metabase/metabase/blob/35d9a8177edd2885bcfa95107c34201c9691bf35/frontend/src/metabase/query_builder/components/template_tags/TagEditorParamParts/DefaultRequiredValueControl.tsx#L30)).

Also the parens are decorative and should not be a part of the translation string.